packages/drivers/routerlicious-driver/package.json

```diff
@@ -70,6 +71,7 @@
     "debug": "^4.1.1",
     "isomorphic-ws": "^4.0.1",
     "json-stringify-safe": "5.0.1",
+    "jsrsasign": "^10.0.2",
```

Comment:
You might already be planning to revert this per @anthony-murphy's comment a couple days ago above, but just wanted to hop in to say the r11s-driver shouldn't need a dependency on jsrsasign. The TokenProvider provided by the host should have all the token signing implementation details

Reply:
yeah, this work was forked off of the tinylicious-client package. there will be some cleanup like this coming

Reply:
oh...that dep is definitely leftover from a prev commit...i'll remove it